resources/responses.js

```diff
@@ -584,36 +597,43 @@ let Responses = {
   },
   sleep: (sev) => {
     let obj = {};
-    obj[defaultAlertText(sev)] = {
-      en: 'Sleep',
-      de: 'Schlaf',
-      fr: 'Sommeil',
-      ja: 'スリプル',
-      cn: '催眠',
-      ko: '슬리플',
+    obj[defaultAlertText(sev)] = (data, matches) => {
+      let source = getSource(matches);
+      return {
+        en: 'Sleep => ' + source,
```

Comment:
At least for English, I think all of these would be better without `=>`

Reply:
yeah it was just to keep the old text (as i am also not sure how other language really handels it)